generator/config/stage/project.yaml

```diff
@@ -11,7 +11,7 @@ arguments:
         name: specification
         type:
             - expression
-        variadic: object
+        variadic: object # XXX: This should somehow allow explicit typings for { fieldName: 1|0 }
```

Comment:
The type is "expression", because any expression is accepted to compute a value.